packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -38,7 +63,49 @@ const CFDsListing = () => {
                 </div>
             )}
 
-            <div className='cfds-listing'>CFD Listing</div>
+            <div className='cfd-full-row' style={{ paddingTop: '2rem' }}>
+                <Text size='xs' line_height='m' weight='bold'>
+                    {localize('Deriv MT5')}
+                </Text>
+            </div>
+            {available_mt5_accounts?.map((account: AvailableAccount) => {
+                return (
+                    <TradingAppCard
+                        {...account}
+                        key={`trading_app_card_${account.name}`}
+                        type={
+                            checkForExistingAccounts(account.platform, account.market_type) ? 'transfer_trade' : 'get'
+                        }
+                        availability={selected_region}
+                        onAction={() => {
+                            startTrade(account.platform, account);
+                        }}
+                    />
+                );
+            })}
+            <div className='cfd-full-row'>
+                <hr className='divider' />
+            </div>
+            {available_dxtrade_accounts?.length > 0 && (
+                <div className='cfd-full-row'>
+                    <Text size='xs' line_height='m' weight='bold'>
+                        {localize('Other CFDs')}
+                    </Text>
+                </div>
+            )}
+            {available_dxtrade_accounts?.map((account: AvailableAccount) => {
+                return (
+                    <TradingAppCard
+                        {...account}
+                        key={`trading_app_card_${account.name}`}
+                        type={
+                            checkForExistingAccounts(account.platform, account.market_type) ? 'transfer_trade' : 'get'
+                        }
+                        availability={selected_region}
+                        is_disabled={is_real ? !has_no_real_account : account.is_disabled}
```

Comment:
If user has no real account, should we disable the dxtrade Get button?
In my understanding,  Get button should be enabled and direct to "You will need a deriv account" modal.

Reply:
I'll look into this, as most of this code is directly translated from `pre_appstore` or `trader's hub`

Reply:
@amina-deriv, I checked and the requirements were changed in DIEL. But this can be removed when the modal is being integrated.